src/cache.jl

```diff
@@ -94,7 +94,11 @@ end
 updatecache(absentmodule::Symbol, server) = updatecache([absentmodule], server)
 
 function updatecache(absentmodules::Vector{Symbol}, server)
+    server.cp_lock && return
```

Comment:
Ah, I didn't mean a persistent child process, I just meant a persistent thread in the main ls process that essentially works on a queue of tasks to process, and then still starts individual julia child processes. It would just serialize the whole thing.